package.json

```diff
@@ -199,5 +195,14 @@
       "prettier --write",
       "git add"
     ]
+  },
+  "jest": {
+    "rootDir": "src",
+    "modulePaths": [
+      "<rootDir>/"
+    ],
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js"
+    }
   }
 }
```

Comment:
None of the versions are `^`. In #2235 they were.

Reply:
correct; we're moving to pinned dependency versions and this is one step in that direction